client/app/bundles/course/assessment/pages/AssessmentShow/AssessmentShowPage.tsx

```diff
@@ -0,0 +1,187 @@
+import { InsertDriveFile } from '@mui/icons-material';
+import {
+  Alert,
+  Chip,
+  List,
+  ListItem,
+  ListItemIcon,
+  ListItemText,
+  Paper,
+  Typography,
+} from '@mui/material';
+import { AssessmentData } from 'types/course/assessment/assessments';
+
+import Subsection from 'lib/components/core/layouts/Subsection';
+import Link from 'lib/components/core/Link';
+import useToggle from 'lib/hooks/useToggle';
+import useTranslation from 'lib/hooks/useTranslation';
+
+import translations from '../../translations';
+
+import DeleteAssessmentPrompt from './prompts/DeleteAssessmentPrompt';
+import AssessmentDetails from './AssessmentDetails';
+import AssessmentShowHeader from './AssessmentShowHeader';
+import NewQuestionMenu from './NewQuestionMenu';
+import QuestionsManager from './QuestionsManager';
+import UnavailableAlert from './UnavailableAlert';
+
+interface AssessmentShowPageProps {
+  for: AssessmentData;
+}
+
+const AssessmentShowPage = (props: AssessmentShowPageProps): JSX.Element => {
+  const { for: assessment } = props;
+  const { t } = useTranslation();
+  const [deleting, toggleDeleting] = useToggle();
+
+  return (
+    <main className="space-y-5">
```

Comment:
These are called [semantic elements](https://developer.mozilla.org/en-US/docs/Glossary/Semantics) in HTML.

`main`, `section`, `article`, `aside`, `nav`, `header`, etc. are all essentially similar to `div`. But semantically, they define different _roles_ of that `div`.

We use `nav` in our sidebar (see `layouts/course.html.slim:22`), for example, because that `div` is a `nav`igation element. Same here. I used `main` to define the main part of this page, and `section` to define sections in the page. This is useful:
- for SEO and screen readers to correctly identify different parts of the site (not really useful for us in this case), and
- for us developers to navigate through the code. Imagine if we have all `div`s, then it will be hellish to read the nested `div`s with all `</div>`s after `</div>`s.

Moving forward, we should make use of these elements for our site, e.g., 
- `header` and `footer`, `aside` for sidebars, 
- `nav` for navigation elements and/or breadcrumbs, 
- `time` for timestamps, 
- `article` for comment/forum posts, 
- etc.